components/Tab/VerticalTabBarItem.vue

```diff
@@ -27,19 +27,19 @@ import tabBarItemMixin from '~/mixins/tabBar/tabBarItemMixin';
 
 export default {
     name: 'VerticalTabBarItem',
-    components: {
-        Icon: () => import('~/components/Icon/Icon'),
-    },
     mixins: [tabBarItemMixin],
     computed: {
-        tabIcon() {
+        tabIconComponent() {
+            return () => import(`~/components/Icon/${this.item.icon}`);
```

Comment:
What if we don't have value for`this.item.icon`. 
Below you check if it exists. 
`if (!this.item.icon) {
                return null;
            }`